### DISCUSSION TITLE
Where can I find for subscription on mailing list or on google groups of MDanalysis?

### DISCUSSION THREAD

**Discussion by g00dtw0** [MDAnalysis/mdanalysis#4461] [Q&A] [answered]
I would like to be joining the mailing list and google group of MDanalysis.

**Comment by RMeli** [comment, marked as answer]
Hello! 
The mailing list (Google group) has been retired, in favour of GitHub Discussions.

The old mailing list messages are still available (see pinned announcements here), but there will be no new messages there.

**Reply by RMeli** [reply]
For reference: https://github.com/MDAnalysis/mdanalysis/discussions/4356

**Comment by g00dtw0** [comment]
Thank you for kind reply.
